### DISCUSSION TITLE
progress bar and slider

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#15] [Ideas]
hello Renaud,

For my 7GUI tasks challenge I need a progress bar and a slider.  As progress bars and slider are pretty common I thought they should be a nice enhancement to dlite..
![image](https://user-images.githubusercontent.com/22573455/169288053-554f7077-dc5f-478a-94d4-497a5c89630f.png)

**Comment by renaudpawlak** [comment]
Nice. The slider is already there. Create a regular input and select the 'range' type.
I don't think I have supported the progress bar. Will do it ASAP (it should be fast, it is a very simple component). Maybe you can use another range in the meantime?

**Reply by HFolkertH** [reply]
Thanks. I will try the input_range.

**Reply by renaudpawlak** [reply]
I have added a progress component today :) It should do the job. Tell me about it whenever you can. Thanks.

**Reply by HFolkertH** [reply]
hi Renaud,

The progress bar works fine.  Thanks. Only the Label property is missing.
